main/musl/APKBUILD

```diff
@@ -119,25 +119,29 @@ utils() {
 
 compat() {
 	pkgdesc="compatibility libraries for glibc"
+	_links="lib/libc.so.6 lib/libcrypt.so.1 lib/libm.so.6 lib/libpthread.so.0 lib/librt.so.1 lib/libutil.so.1"
```

Comment:
Should be "local _links=...".
But in fact I would have preferred to not touch the `for` loop. Just keep `_ld` and make it have the path prefix. This would make the patch less intrusive.
But other than this minor issue looks good to me.